upup/models/cloudup/resources/addons/karpenter.sh/k8s-1.19.yaml.template

```diff
@@ -623,9 +679,8 @@ data:
         "timeEncoder": "iso8601"
       }
     }
-  # Log level overrides
-  loglevel.controller: debug
-  loglevel.webhook: debug
+  loglevel.webhook: "error"
```

Comment:
Since we frequently get asked about debug logs, this probably should be configurable if we move to using error.

Reply:
I prefer to re-set it to debug mode and propose an other future PR to be configurable to not overcharge this PR
 - https://github.com/kubernetes/kops/pull/15144/commits/c08c7b5ef17e8b7f4095b90ddf2b462914370a2a